speech-commands/training/browser-fft/tflite_conversion.ipynb

```diff
@@ -0,0 +1,346 @@
+{
```

Comment:
<p>please do not use the rc0 version, it is referencing the nightly estimator build.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tfjs-models/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I agree we should avoid release candidates in general. However, in this case I have to use 2.3.0rc0. This is the early non-nightly version in which this tensorflow bug is fixed: <a href="https://github.com/tensorflow/tensorflow/issues/38135" target="_blank">https://github.com/tensorflow/tensorflow/issues/38135</a></p><p> </p><p>When the final release of 2.3.0 comes out, I'll switch to that version.</p>

Reply:
please add a TODO for this, afaik the rc build is broken somewhat too.

Reply:
Done.